recipes-bsp/u-boot/README.md

```diff
@@ -59,6 +59,15 @@ apt-get install flashrom
 
 Attach the grabber to the flash chip, then run the following on the host machine:
 
+if the the size of the uboot file doesn't match the flash chip's size (16777216 B)
```

Comment:
Nit: "If..." (capitalization at the beginning of a sentence) - and maybe write "(16 MB)" here.

And could you squash the second commit into the first one (git rebase...)? Thanks.

Reply:
I have completed the modification, please check.
Thanks.